Doc/c-api/unicode.rst

```diff
@@ -1061,6 +1061,28 @@ These are the UTF-8 codec APIs:
    raised by the codec.
 
 
+.. c:function: int PyUnicode_GetUTF8Buffer(PyObject *unicode, const char errors, Py_buffer *view)
```

Comment:
I am not sure about the order of parameters. There is a logic in placing the output parameter at the end, but in `PyObject_GetBuffer()` (for which I modelled the name) it is not the last parameter. Also, if we will add more similar functions in future, with multiple additional parameters (like `flags` and `errors`), it is better to place them at the end.

Reply:
I like having view at the end.